### PR TITLE
Dev/login screen

### PR DESCRIPTION
**Pull request by eapostol**

### CODE REVIEW

File: index.html
Comment (eapostol) [inline index.html:1]: Just note near the end of the project we may wish to minify the code (remove white spaces) to improve delivery and rendering performance.

File: js/main.js
Comment (eapostol) [inline js/main.js:1]: in the future, maybe start by making the code a little readable with comments. then we can clean it up after.

**Review by eapostol — COMMENTED** [review]
I have approved this PR. the code looks good. we will clean it up near the end of the project. I will merge these changes now to the main branch.

File: LoginPage.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Login Page
```
[the remark this answers is not in the document]
Comment (eapostol-2u) [reply LoginPage.md:1]: at @eapostol will do! thanks!

File: index.html
Comment (eapostol-2u) [reply index.html:1]: @eapostol for sure!

**Review by eapostol-2u — APPROVED** [review]
Hi there! I have approved the changes and the review made by @eapostol (the other developer in this project! )

**Pull request merged**